### PR TITLE
AVX-26997 update terraform aviatrix OCI module source url and output …

### PR DESCRIPTION
**Pull request by MingweiZhu666**
…description

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The following example will launch aviatrix_controller_build module,  aviatrix_co
 ```
 # aviatrix_controller_build
 module "aviatrix_controller_build" {
-  source           = "./modules/aviatrix-controller-build"
+  source           = "github.com/AviatrixSystems/terraform-module-oci.git//modules/aviatrix-controller-build"
```
Comment (inc1t3Ful) [inline example/README.md]: these source links are actually incorrect. You should now be sourcing from the registry

File: example/README.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ module "aviatrix_controller_build" {
 
 # aviatrix_controller_initialize
 module "aviatrix_controller_initialize" {
-  source                        = "./modules/aviatrix-controller-initialize"
+  source                        = "github.com/AviatrixSystems/terraform-module-oci.git//modules/aviatrix-controller-initialize"
```
Comment (inc1t3Ful) [inline example/README.md]: See above, need to source from registry instead of github

**Review by inc1t3Ful — CHANGES_REQUESTED** [review]
Please update the READMEs in the submodules as well, to match the format of the main README

File: example/README.md (hunk 1 above)
Comment (MingweiZhu666) [reply example/README.md]: yes, I just find it, fix now. thank you

**Comment by MingweiZhu666** [comment]
> Please update the READMEs in the submodules as well, to match the format of the main README

have fixed

File: modules/aviatrix-controller-initialize/README.md [hunk 3]
```diff
@@ -1,75 +1,30 @@
...
+|   avx_controller_private_ip   |        Aviatrix controller private IP address.         |
+|  avx_controller_admin_email   |        Aviatrix controller admin email address.        |
+| avx_controller_admin_password |          Aviatrix controller admin password.           |
+|        oci_tenancy_id         |                     Tenancy OCID.                      |
+|          oci_user_id          |                       User OCID.                       |
+|      oci_compartment_id       |                   Compartment OCID.                    |
+|       oci_api_key_path        |                   API key file path.                   |
+|         account_email         |       Aviatrix controller access account email.        |
+|      access_account_name      |        Aviatrix controller access account name.        |
+|     aviatrix_customer_id      |             Aviatrix customer license id".             |
```
Comment (inc1t3Ful) [inline modules/aviatrix-controller-initialize/README.md]: remove the random trailing " quotation

**Review by inc1t3Ful — APPROVED** [review]
One more reminder to double check your examples, in case you still have sensitive data leftover, before merging

**Review by inc1t3Ful — CHANGES_REQUESTED** [review]
see above- accidentally hit approve

**Comment by MingweiZhu666** [comment]
> One more reminder to double check your examples, in case you still have sensitive data leftover, before merging

have checked and fixed, thank you

**Review by inc1t3Ful — APPROVED** [review]

**Review by zhiqiangjin — APPROVED** [review]

**Pull request merged**